### PR TITLE
Hunter Smith - DOM -II

### PR DESCRIPTION
**Pull request by Hunter315**

### CODE REVIEW

**Comment by Hunter315** [comment]
@kamry-bowman

File: js/index.js [hunk 1]
```diff
@@ -1 +1,112 @@
...
+/*===================Event listeners===================*/
+
+//click on fun bus logo, changes color
+funBusMain.addEventListener('click', () =>{
+    funBusMain.style.color = 'darkkhaki';
+});
+
+//Nav elements mouseover Grow and Color Change
+for (let i = 0; i < nav.length; i++){
+    nav[i].addEventListener('mouseover', () =>{
```
Comment (kamry-bowman) [inline js/index.js]: nice use of loops!

File: js/index.js [hunk 2]
```diff
@@ -1 +1,112 @@
...
+for (let i = 0; i < nav.length; i++){
+    nav[i].addEventListener('mouseout', (event) =>{
+        nav[i].style.fontSize = '1.6rem';
+        nav[i].style.color = 'black';
+    });
+}
+
+//mouse over nav makes it blue
+mainNav.addEventListener('mouseover', (event) => {  
+    event.preventDefault();
```
Comment (kamry-bowman) [inline js/index.js]: if there isn't a default for mouseover (I don't think there is in this case), you don't need to use event.preventDefault

File: js/index.js [hunk 3]
```diff
@@ -1 +1,112 @@
...
+    if (event.key === 'n') {
+        body.style.backgroundColor = 'rgb(71, 71, 73';
+        mainNav.style.background = 'rgb(50, 50, 52)';
+        footer.style.background = 'rgb(50, 50, 52)';
+        anchorText.forEach((element) => {element.style.color = 'white';});
+        paragraphText.forEach((element) => {element.style.color = 'white';});
+        h1.style.color = 'white';
+        h2.forEach((element) => {element.style.color = 'white';});
+        h4.forEach((element) => {element.style.color = 'white';});
+    }
```
Comment (kamry-bowman) [inline js/index.js]: super cool functionality!

File: js/index.js [hunk 4]
```diff
@@ -1 +1,112 @@
...
+});
+
+//scrolling changes background
+window.addEventListener('scroll', function() {
+    event.stopPropagation();
```
Comment (kamry-bowman) [inline js/index.js]: window is the top element, so you don't need to stop propagation in it

File: js/index.js [hunk 5]
```diff
@@ -1 +1,112 @@
...
+});
+
+//clicking footer makes it white.
+footer.addEventListener('click', function(){ 
+    this.style.background = 'white';
+});
+
+//double clicking body reloads the document
+body.addEventListener('dblclick', function(){ 
+    location.reload();
```
Comment (kamry-bowman) [inline js/index.js]: interesting, didn't know this method

File: js/index.js [hunk 6]
```diff
@@ -1 +1,112 @@
...
+    // e.preventDefault();
+    // e.stopPropagation();
+    // body.style.backgroundImage = "url(img/fun-bus.jpg)";
+// })
+
+//button color change animation with stop propagation.
+button.forEach((element) => {  
+    element.addEventListener('click', (event) => {
+        event.stopPropagation();
+        if (event.target.style.backgroundColor != 'blue'){
```
Comment (kamry-bowman) [inline js/index.js]: careful when using != and not !==, != cannot sometimes behave oddly

**Pull request closed without merging**